bundler.d/assets.rb

```diff
@@ -10,8 +13,8 @@
 gem 'execjs', '>= 1.4.0', '<2.5.0'
 gem 'ace-rails-ap', '~> 4.0.0'
 gem 'flot-rails', '0.0.3'
-gem 'sass-rails', '~> 4.0'
 gem 'uglifier', '>= 1.0.3'
+gem 'sass-rails', '~> 5.0.4'
```

Comment:
`~> 5.0` ought to be fine, this is pinning it to `5.0.9` max, minor versions shouldn't break compatibility. Same above for sprockets-rails